src/main/java/uk/gov/pay/ledger/event/model/SalientEventType.java

```diff
@@ -16,6 +16,7 @@
     UNEXPECTED_GATEWAY_ERROR_DURING_AUTHORISATION,
     STATUS_CORRECTED_TO_AUTHORISATION_ERROR_TO_MATCH_GATEWAY_STATUS,
     GATEWAY_REQUIRES_3DS_AUTHORISATION,
+    GATEWAY_3DS_EXEMPTION_RESULT_OBTAINED,
```

Comment:
Modelling this in Ledger is only needed if this event will map to an external state transition for the entity, this allows the public API responses to be guaranteed consistent independent of consumers emitting events.

For events that don't need to transition the external state of their entity, Ledger can happily parse any name/ data combination without it being explicitly defined.

Not sure of what the use case for the proposed event is so just double checking!

Reply:
Ha good spot! Removed that changed so its all about refactoring the lambdas into method references.